Hast.Abstractions/Hast.Vitis.Abstractions/Services/ZynqHardwareImplementationComposerBuildProvider.cs

```diff
@@ -89,9 +92,11 @@ public sealed class ZynqHardwareImplementationComposerBuildProvider
         await _buildLogger.ExecuteWithLoggingAsync(vivadoExecutable, vivadoArguments, tmpDirectoryPath);
         MajorProgress("Frequency scaling profile created.");
 
+        const string input = "--input";

```

Comment:
How come you don't need the same for "--force" and "--info" too?

Reply:
The analyzer's threshold seems to be at 3, so violation for `"--input"` but not for the others. However, thinking about it, this actually makes the code less readable, so I'm suppressing instead.